### PR TITLE
Add ability for KerasClassifier to process string classes.

### PR DESCRIPTION
**Pull request by reiinakano**
Scikit-learn classifiers are able to process strings in their target classes. This PR allows KerasClassifier to do the same while also exposing a `.classes_` attribute containing the classes used during fitting.

I modified the corresponding unit tests to test the behavior of KerasClassifier when fitted with string labels.

### CODE REVIEW

**Comment by fchollet** [comment]
Closing outdated PR. If you still care about the content of the PR, please submit a new PR to `master`, updated for the Keras 2.0 API.
